tests/common/devices/sonic.py

```diff
@@ -890,7 +900,7 @@ def shutdown(self, ifname):
                 ifname: the interface to shutdown
         """
         logging.info("Shutting down {}".format(ifname))
-        return self.command("sudo config interface shutdown {}".format(ifname))
+        return self.command("sudo config interface -n \"{}\" shutdown {}".format(self._get_namespace_for_port(ifname), ifname))
```

Comment:
this might fail for single asic ? there will be additonal -n in the command ?

Reply:
can we try to use these functions
https://github.com/Azure/sonic-mgmt/blob/003a37cf6315eb044722bfaf5ad58e8ad9992152/tests/common/devices/sonic_asic.py#L471-L478

Reply:
Thank you, I have modified as suggested.

Reply:
> this might fail for single asic ? there will be additonal -n in the command ?

This would not fail as -n "" will work for single asic as well.